client/my-sites/hosting/staging-site-card/index.js

```diff
@@ -227,14 +252,24 @@ const StagingSiteCard = ( { currentUserId, disabled, siteId, siteOwnerId, transl
 	);
 };
 
-export default connect( ( state ) => {
-	const currentUserId = getCurrentUserId( state );
-	const siteId = getSelectedSiteId( state );
-	const siteOwnerId = getSelectedSite( state )?.site_owner;
+export default withMediaStorage(
```

Comment:
What is the benefit of using this wrapper compared to `useMediaStorageQuery`?

Reply:
None. I just used it as we have already this component connected to the state, and `withMediaStorage` was used the same way as I use it here, somewhere else, so I thought to reuse that.